### PR TITLE
Removed need for Heroku to re-compile Monit binary on each push

### PR DESCRIPTION
**Pull request by mefellows**
Also included SSL support and upgraded to 5.8

### CODE REVIEW

**Comment by mefellows** [comment]
Any updates on this PR?

File: README.md [hunk 1]
```diff
@@ -1,11 +1,15 @@
-# Heroku Buildpack: Monit
+# Heroku Buildpack: Monit 5.8
```
Comment (matiaskorhonen) [inline README.md:1]: There's really no need to add the version number to the title…

File: README.md [hunk 2]
```diff
@@ -1,11 +1,15 @@
...
 
 Run [Monit](http://mmonit.com/monit/) on Heroku.
 
-Based on the [null-buildpack][null-buildpack] by [Ryan Smith][ryandotsmith].
+Based on the [monit-buildpack][monit-buildpack] by [Matias Korhonen][matiaskorhonen].
```
Comment (matiaskorhonen) [inline README.md:5]: I'm guessing this line got pulled into the PR by mistake?

File: README.md [hunk 3]
```diff
@@ -1,11 +1,15 @@
...
 
-[null-buildpack]: https://github.com/ryandotsmith/null-buildpack
-[ryandotsmith]: https://github.com/ryandotsmith
+[monit-buildpack]: https://github.com/matiaskorhonen/monit-buildpack
+[matiaskorhonen]: https://github.com/matiaskorhonen
+
+# What is different from Upstream?
+
+The main difference is that this allows you to modify the Monit configuration without requiring a full re-compilation of Monit on push. I found that I was making many changes and didn't like waiting several minutes for the system to re-compile. This works around that by caching the binaries between builds (this is a relatively safe thing to do, since the version of the binaries and underlying platform won't change for you between builds).
```
Comment (matiaskorhonen) [inline README.md:12]: Once this is pulled in, this will be the upstream so this section will be redundant.

**Comment by matiaskorhonen** [comment]
Sorry, I seem to have completely forgotten about this PR.

I added a couple of comments since there were some oddities in the README changes, but if you fix those and squash your commits I'll merge this in, I promise!

(you may need to update against master to get the PR to merge cleanly)

**Comment by mefellows** [comment]
As mentioned previously, I will resubmit a fresh PR with above taken into consideration.

**Pull request closed without merging**